src/components/Switch/index.tsx

```diff
@@ -245,40 +306,18 @@ Switch.propTypes = {
   checked: PropTypes.bool,
   disabled: PropTypes.bool,
   id: PropTypes.string,
-  /**
-   * Left and right will be outside of switch button in the contrary of inside value.
-   */
   labeled: PropTypes.oneOfType([
     PropTypes.bool,
-    PropTypes.oneOf(['inside', 'left', 'right']),
+    PropTypes.oneOf(labelPositions),
   ]),
   name: PropTypes.string.isRequired,
-  /**
-   * Text or node shown in switch button when its state if off
-   */
   offLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
   onChange: PropTypes.func.isRequired,
-  /**
-   * Text or node shown in switch button when its state if on
-   */
```

Comment:
Those text are supposed to be shown in Storybook, can you keep them and move them into type definitions ?